src/Constants.h

```diff
@@ -94,6 +94,17 @@ constexpr const char* ARDUPILOT_PROTOCOL_NAME = "/ardupilot";
 constexpr std::chrono::milliseconds JOINT_POWER_REPEAT_PERIOD(333);
 constexpr std::chrono::milliseconds ARM_IK_UPDATE_PERIOD(50);
 
+constexpr std::array<robot::types::motorid_t, 4> WHEEL_IDS = {
+	robot::types::motorid_t::frontLeftWheel,
+	robot::types::motorid_t::frontRightWheel,
+	robot::types::motorid_t::rearLeftWheel,
+	robot::types::motorid_t::rearRightWheel,
+};
+constexpr std::array<int32_t, 4> NORMAL_WHEEL_ROTS = {0, 0, 0, 0};
+constexpr std::array<int32_t, 4> TURN_IN_PLACE_WHEEL_ROTS = {45000, -45000, 45000, -45000};
+constexpr std::array<int32_t, 4> CRAB_WHEEL_ROTS = {90000, 90000, 90000, 90000};
```

Comment:
These should be calculated from the swerve kinematics, not hardcoded, since the angles will change based on the wheelbase dimensions.